precovery/main.py

```
@@ -22,7 +23,10 @@ def precover_many(
     allow_version_mismatch: bool = False,
     datasets: Optional[set[str]] = None,
     n_workers: int = multiprocessing.cpu_count(),
-) -> Tuple[PrecoveryCandidatesQv, FrameCandidatesQv]:
+    propagator: Optional[
+        Propagator
+    ] = None,  # should we initialize an assist propagator?
```

Not sure. We should probably just throw an error for now.